src/stateAggregator/accessors/aliasAccessor.ts

```diff
@@ -148,23 +203,87 @@ export class AliasAccessor extends AsyncOptionalCreatable {
    * @param alias the alias you want to check
    */
   public has(alias: string): boolean {
-    return this.config.has(alias);
+    return this.aliasStore.has(alias);
   }
 
   protected async init(): Promise<void> {
-    this.config = await AliasesConfig.create(AliasesConfig.getDefaultOptions());
+    this.fileLocation = getFileLocation();
+    await this.readFileToAliasStore();
   }
 
   /**
-   * Returns the username of given aliasable entity
+   * go to the fileSystem and read the file, storing a copy in the class's store
+   * if the file doesn't exist, create it empty
    */
-  // eslint-disable-next-line class-methods-use-this
-  private getNameOf(entity: Aliasable): string {
-    if (typeof entity === 'string') return entity;
-    const aliaseeName = entity.username ?? entity.user;
-    if (!aliaseeName) {
-      throw new SfError(`Invalid aliasee, it must contain a user or username property: ${JSON.stringify(entity)}`);
+  private async readFileToAliasStore(useLock = false): Promise<void> {
+    try {
+      if (useLock) {
+        await lock(this.fileLocation, lockRetryOptions);
+      }
+      this.aliasStore = fileContentsRawToAliasStore(await readFile(this.fileLocation, 'utf-8'));
+    } catch (e) {
+      if (e instanceof Error && 'code' in e && e.code === 'ENOENT') {
+        this.aliasStore = new Map<string, string>();
+        await mkdir(dirname(this.fileLocation), { recursive: true });
+        await this.saveAliasStoreToFile();
+        return;
+      }
+      if (useLock) await unlock(this.fileLocation);
```

Comment:
This is an edge case, but seems possible:
`lock` throws an error that is not an ENOENT (maybe an EPERM error) so the lock is never obtained.  In the catch we try to unlock and that throws an error, and is the one reported to the user.  The user wouldn't know why the lock couldn't be obtained.  To prevent this, we might need to use a try/catch/finally and the finally has a try/catch for the unlock.

Reply:
I think putting the original `lock` outside the try should surface an error better.